Patch to remove 3rd party software and replace lapack(e) with openblas
Author: J. Sassmannshausen NHS/GSTT 
diff --git a/regenie-2.2.4-orig/Makefile b/regenie-2.2.4/Makefile
index 03c56e4..51104d9 100755
--- a/regenie-2.2.4-orig/Makefile
+++ b/regenie-2.2.4/Makefile
@@ -45,7 +45,7 @@ endif
 # detect OS architecture and add flags
 UNAME_S      := $(shell uname -s)
 ifeq ($(UNAME_S),Linux)
- INC          = -I${BGEN_PATH}/3rd_party/boost_1_55_0
+ INC          = -I${BOOST}
  CFLAGS      += -fopenmp
  ifeq ($(strip $(STATIC)),1)
   LPATHS      = -static-libgcc -static-libstdc++
@@ -104,11 +104,11 @@ else ifneq ($(strip $(OPENBLAS_ROOT)),)
   INC        += -I${OPENBLAS_ROOT}/include/
   # static linking
   ifeq ($(strip $(STATIC)),1)
-   SLIBS     += -Wl,-rpath=${OPENBLAS_ROOT}/lib/ -llapack -llapacke -lopenblas
+   SLIBS     += -Wl,-rpath=${OPENBLAS_ROOT}/lib/ -lopenblas
    DLIBS     += -lgfortran
   # dynamic linking
   else
-   DLIBS     += -Wl,-rpath=${OPENBLAS_ROOT}/lib/ -llapack -llapacke -lopenblas -lgfortran
+   DLIBS     += -Wl,-rpath=${OPENBLAS_ROOT}/lib/ -lopenblas -lgfortran
   endif
  endif
 endif
@@ -121,11 +121,11 @@ RGFLAGS      += -DVERSION_NUMBER=\"$(RG_VERSION)\"
 OBJECTS       = $(patsubst %.cpp,%.o,$(wildcard ./src/*.cpp))
 
 PGEN_PATH     = ./external_libs/pgenlib/
-INC          += -I${PGEN_PATH} -I${PGEN_PATH}/simde/ -I${PGEN_PATH}/include/ -I./external_libs/cxxopts/include/ -I./external_libs/LBFGSpp/include/ -I${BGEN_PATH} -I./external_libs/eigen3/ -I${BGEN_PATH}/genfile/include/ -I${BGEN_PATH}/3rd_party/zstd-1.1.0/lib -I${BGEN_PATH}/db/include/ -I${BGEN_PATH}/3rd_party/sqlite3 -I./external_libs/
+INC          += -I${PGEN_PATH} -I${PGEN_PATH}/simde/ -I${PGEN_PATH}/include/ -I./external_libs/cxxopts/include/ -I./external_libs/LBFGSpp/include/ -I./external_libs/eigen3/ -I${BGEN_PATH}/include/ -I${BOOST}/include -I./external_libs/
 
-LPATHS       += ${LIBMKL} -L${BGEN_PATH}/build/ -L${BGEN_PATH}/build/3rd_party/zstd-1.1.0/ -L${BGEN_PATH}/build/db/ -L${BGEN_PATH}/build/3rd_party/sqlite3/ -L${BGEN_PATH}/build/3rd_party/boost_1_55_0 -L/usr/lib/
+LPATHS       += ${LIBMKL} -L${BGEN_PATH}/lib -L${BOOST}/lib 
 
-LIBS         += ${SLIBS} -lbgen -lzstd -ldb  -lsqlite3 -lboost
+LIBS         += ${SLIBS} -lbgen -lzstd -ldb  -lsqlite3 -lboost_system -lboost_filesystem -lboost_thread -lboost_timer
 LIBS         += -lz ${DLIBS} -lm -ldl
 
 
